src/metabase/automagic_dashboards/core.clj

```diff
@@ -702,7 +702,8 @@
                   (format "%s#show=all" url))
           :transient_filters query-filter
           :param_fields (filter-referenced-fields root query-filter)
-          :auto_apply_filters true))))
+          :auto_apply_filters true
+          :width "fixed"))))
```

Comment:
Looked into it, and that’s as good a place as any to add the key.
One difference here, just to keep a good mental model, is that automagic dashboards are ephemeral. We are not pulling anything from the AppDB, and (as far as I understand it), once you hit ‘save’, it’s just a normal dashboard. So, the key you added is part of generating this ‘ephemeral’ dashboard, not requesting an additional column from the db.

The embed change we made pulls already existing data from the AppDB.

This doesn’t affect your change in any way, I just wanted to explain the difference between the embedded dashboard API and the automagic Dashboards.
I’ll add this as a review on the PR too. :slightly_smiling_face: